avaframe/in1Data/getInput.py

```diff
@@ -65,59 +65,59 @@ def getDEMPath(avaDir):
     """
 
     # if more than one .asc file found throw error
-    inputDir = pathlib.Path(avaDir, 'Inputs')
-    demFile = list(inputDir.glob('*.asc'))
+    inputDir = pathlib.Path(avaDir, "Inputs")
+    demFile = list(inputDir.glob("*.asc"))
     if len(demFile) > 1:
-        message = 'There should be exactly one topography .asc file in %s/Inputs/' % (avaDir)
+        message = "There should be exactly one topography .asc file in %s/Inputs/" % avaDir
         log.error(message)
         raise AssertionError(message)
 
     # if is no .asc file found - throw error
-    filesFound = list(inputDir.glob('*.*'))
+    filesFound = list(inputDir.glob("*.*"))
     if len(demFile) == 0 and len(filesFound):
         for fileF in filesFound:
-            message = 'DEM file format not correct in %s/Inputs/ - only .asc is allowed but %s is provided' % (avaDir, fileF.name)
+            message = (
+                "DEM file format not correct in %s/Inputs/ - only .asc is allowed but %s is provided"
+                % (avaDir, fileF.name)
+            )
             log.error(message)
             raise AssertionError(message)
     elif len(demFile) == 0:
-        message = 'No topography .asc file in %s/Inputs/' % (avaDir)
+        message = "No topography .asc file in %s/Inputs/" % avaDir
         log.error(message)
         raise FileNotFoundError(message)
 
     return demFile[0]
 
 
-def getDEMFromConfig(avaDir, fileName=''):
-    """ get dem file path in avaDir/Inputs or if fileName in avaDir/Inputs/fileName
+def getDEMFromConfig(avaDir, fileName=""):
+    """get dem file path in avaDir/Inputs or if fileName in avaDir/Inputs/fileName
 
-        Parameters
-        -----------
-        avaDir: str or pathlib path
-            path to avalancheDir
-        fileName: pathlib path
-            path to dem file with filename in avaDir/Inputs
+    Parameters
+    -----------
+    avaDir: str or pathlib path
+        path to avalancheDir
+    fileName: pathlib path
+        path to dem file with filename in avaDir/Inputs
 
-        Returns
-        --------
-        demFile: pathlib path
-            path to dem file
+    Returns
+    --------
+    demFile: pathlib path
+        path to dem file
     """
 
-    if fileName == '':
-        demFilePath = getDEMPath(avaDir)
```

Comment:
isn't this option lost in the new code? that if no fileName provided it looks for the dem in Inputs?

Reply:
demFilePath is never used, so this if makes no sense at all and only leads to an empty / undefined demFile being returned. Or am I missing something here?

Reply:
ok, I'll include this then and make a pytest for it. Since we obviously missed it so far :-)